x/leverage/types/genesis.go

```diff
@@ -1,16 +1,25 @@
 package types
 
+import fmt "fmt"
+
 // DefaultGenesis returns the default genesis state of the x/leverage module.
 func DefaultGenesis() *GenesisState {
 	return &GenesisState{
-		Params:   Params{},
+		Params: Params{
+			InterestEpoch: 100,
+		},
 		Registry: []Token{},
 	}
 }
 
 // Validate performs basic genesis state validation returning an error upon any
 // failure.
 func (gs GenesisState) Validate() error {
+
+	if gs.Params.InterestEpoch <= 0 {
+		return fmt.Errorf("interest epoch must be positive: %d", gs.Params.InterestEpoch)
+	}
```

Comment:
We should instead call `params.Validate()`

Reply:
(done)